src/osc-bridge/src/parse-schema.c

```diff
@@ -105,6 +105,19 @@ void parse_range(schema_handle_t *handle, const char *str, int len)
         handle->value_max = atof(tok.str);
     else
         fprintf(stdout, "[WARNING] Unexpected Range Type %d For Max\n", tok.type);
+
+    array = mm_json_read(&tok, &array);
+    if(!array.src) {
+        // this is not an error - this means that the schema does not
+        // support the third "logmin" parameter yet
+        handle->value_logmin = handle->value_min;
```

Comment:
This line here creates an ordering dependency between logmin and the min value in the schema. While it's not currently an issue, I could see this causing a confusing bug down the road.

Reply:
Do you mean that the handles run through a sort routine, which would have different results if a logmin is available vs if not? Or maybe I don't understand what you mean. Can you please elaborate?

Reply:
Nevermind. I totally made a bad assumption based on how this ought to work. After checking the updated schema file, my comment makes zero sense.